src/main/java/org/broadinstitute/hellbender/tools/spark/transforms/markduplicates/MarkDuplicatesSpark.java

```diff
@@ -34,12 +34,62 @@
 
 import java.util.*;
 
+/**
+ * <p>This tool is a Spark implementation of the tool MarkDuplicates in Picard allowing for better utilization
+ *    of available system resources to speed up duplicate marking.</p>
```

Comment:
Can you add a bit more Spark-specific information? For example, a working example Spark command showing how to run the tool on multiple cores, information about memory requirements per core, etc.

Reply:
Also, the wording here could be better. How about this: 

```
This is a Spark implementation of the MarkDuplicates tool from Picard that allows the tool to be run in 
parallel on multiple cores on a local machine or multiple machines on a Spark cluster while still matching 
the output of the single-core Picard version.
```